partner_firstname/partner.py

```diff
@@ -68,10 +68,10 @@ def _write_name(
         In addition an heuristic avoids to keep a firstname without a
         non-blank lastname
         """
-        field_value = (field_value
-                       and not field_value.isspace()
-                       and field_value
-                       or False)
+        field_value = (
+            field_value and not field_value.isspace() and field_value or
+            False
```

Comment:
Could you write?

    field_value if field_value and not field_value.isspace() else False